src/Microsoft.AspNetCore.Identity.EntityFrameworkCore/UserStore.cs

```diff
@@ -1196,5 +1197,69 @@ where userrole.RoleId.Equals(role.Id)
             }
             return new List<TUser>();
         }
+
+        private Task<IdentityUserToken<TKey>> FindToken(TUser user, string loginProvider, string name, CancellationToken cancellationToken)
+        {
+            var userId = user.Id;
+            return UserTokens.SingleOrDefaultAsync(l => l.UserId.Equals(userId) && l.LoginProvider == loginProvider && l.TokenName == name, cancellationToken);
+        }
+
+        // <inheritdoc>
+        public virtual async Task SetTokenAsync(TUser user, string loginProvider, string name, string value, CancellationToken cancellationToken)
```

Comment:
I'm not sure it makes things clearer as...
  - the risk is high to mix them up with 2-FA tokens, that are also "authentication tokens".
  - they are more authorization tokens you can use to query an API or to refresh a token than real authentication tokens.